spec/elastic/transport/opentelemetry_spec.rb

```diff
@@ -0,0 +1,81 @@
+# Licensed to Elasticsearch B.V. under one or more contributor
+# license agreements. See the NOTICE file distributed with
+# this work for additional information regarding copyright
+# ownership. Elasticsearch B.V. licenses this file to you under
+# the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+require 'spec_helper'
+
+if defined?(::OpenTelemetry)
+  describe Elastic::Transport::OpenTelemetry do
+    let(:exporter) { EXPORTER }
+    before { exporter.reset }
+    after { exporter.reset }
+    let(:span) { exporter.finished_spans[0] }
+
+    let(:client) do
+      Elastic::Transport::Client.new.tap do |_client|
+        allow(_client).to receive(:__build_connections)
+      end
+    end
+
+    let(:otel) { described_class.new }
+
+    context 'when path parameters' do
+      it 'creates a span with path parameters' do
+        client.perform_request(
+          'DELETE', '/foo,bar/_aliases/abc,xyz', nil, nil, nil, ["/{index}/_alias/{name}", "/{index}/_aliases/{name}"],
+          'indices.delete_alias'
+        )
```

Comment:
@picandocodigo can you tell me how you start ES so that this request can succeed against a test server?

Reply:
@estolfo For testing locally I use [the docker rake](https://github.com/elastic/elasticsearch-ruby/blob/main/CONTRIBUTING.md#tests) task in `elasticsearch-ruby`.